src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebRequestSession.cs

```diff
@@ -64,6 +64,16 @@ public class WebRequestSession
         /// </summary>
         public int MaximumRedirection { get; set; }
 
+        /// <summary>
+        /// Gets or sets the count of retries for request failures.
+        /// </summary>
+        public int RetryCount { get; set; }
```

Comment:
To be consistent, I wonder if this should be `MaximumRetry`

Reply:
I like this suggestion, I will make this change.

Reply:
`MaximumRetry` sounds weird. Maybe `MaximumRetryCount`?

Looking at existing commands, `New-PSSessionOption` uses `-MaxConnectionRetryCount`

Reply:
Fixed.